lib/wss/cmds/update_ao.js

```diff
@@ -0,0 +1,15 @@
+'use strict'
+
+const debug = require('debug')('bfx:hf:algo-server:wss:cmd:update-ao')
+const send = require('../send')
+
+module.exports = async (as, ws, msg) => {
+  const [, type, params] = msg
+
+  try {
+    await as.host.changeAOStatus(type, params)
+  } catch (e) {
+    send(ws, ['error', e.message])
+    debug('%s', e.stack)
+  }
+}
```

Comment:
make sure you have "new line at end of file" enabled, just to make git a little prettier